rhv4/profiles/rhvh-stig.profile

```diff
@@ -0,0 +1,453 @@
+    - ensure_redhat_gpgkey_installed
+    - ensure_gpgcheck_globally_activated
+    - ensure_gpgcheck_never_disabled
+    - ensure_gpgcheck_repo_metadata
+    - ensure_gpgcheck_local_packages
+    - network_sniffer_disabled
+    - network_ipv6_disable_rpc
+    - network_ipv6_privacy_extensions
+    - sshd_enable_x11_forwarding
+    - clean_components_post_updating
+    - kernel_module_cramfs_disabled
+    - kernel_module_dccp_disabled
+    - kernel_module_freevxfs_disabled
+    - kernel_module_hfs_disabled
+    - kernel_module_hfsplus_disabled
+    - kernel_module_jffs2_disabled
+    - kernel_module_sctp_disabled
+    - var_sssd_ssh_known_hosts_timeout=5_minutes
+    - var_accounts_minimum_age_login_defs=1
+    - var_password_pam_maxrepeat=3
+    - var_account_disable_post_pw_expiration=0
+    - var_removable_partition=dev_cdrom
+    - var_time_service_set_maxpoll=system_default
+    - sysctl_net_ipv4_conf_all_accept_source_route_value=disabled
+    - sysctl_net_ipv6_conf_all_accept_source_route_value=disabled
+    - rpm_verify_ownership
+    - accounts_password_pam_maxrepeat
+    - accounts_password_set_min_life_existing
+    - accounts_password_set_max_life_existing
+    - sudo_remove_nopasswd
+    - sudo_remove_no_authenticate
+    - selinux_user_login_roles
+    - accounts_user_interactive_home_directory_defined
+    - accounts_have_homedir_login_defs
+    - accounts_user_interactive_home_directory_exists
+    - file_permissions_home_directories
+    - file_ownership_home_directories
+    - file_groupownership_home_directories
+    - accounts_users_home_files_ownership
+    - accounts_users_home_files_groupownership
+    - accounts_users_home_files_permissions
+    - accounts_user_dot_user_ownership
+    - accounts_user_dot_group_ownership
+    - file_permission_user_init_files
+    - accounts_user_home_paths_only
+    - accounts_user_dot_no_world_writable_programs
+    - mount_option_home_nosuid
+    - accounts_umask_interactive_users
+    - partition_for_home
+    - partition_for_var
+    - partition_for_var_log_audit
+    - partition_for_tmp
+    - grub2_no_removeable_media
+    - auditd_audispd_configure_remote_server
+    - auditd_audispd_encrypt_sent_records
+    - auditd_audispd_disk_full_action
+    - auditd_data_retention_space_left
+    - audit_rules_execution_setfiles
+    - audit_rules_kernel_module_loading_create
+    - audit_rules_kernel_module_loading_finit
+    - install_mcafee_antivirus
+    - mcafee_antivirus_definitions_updated
+    - var_mcafee_antivirus_definition_expire=1_week
+    - configure_firewalld_ports
+    - sssd_ldap_start_tls
+    - sssd_ldap_configure_tls_ca_dir
+    - sssd_ldap_configure_tls_ca
+    - package_openssh-server_installed
+    - sshd_print_last_log
+    - chronyd_or_ntpd_set_maxpoll
+    - configure_firewalld_rate_limiting
+    - no_user_host_based_files
+    - no_host_based_files
+    - network_configure_name_resolution
+    - postfix_prevent_unrestricted_relay
+    - package_vsftpd_removed
```

Comment:
Are these packages installable on a RHV-H host? Meaning available in the package groups at all?

Reply:
@shawndwells they are for a `RHEL-H` host.

Reply:
@shawndwells I would like to make the `RHVH` and `RHELH` to be as similar as possible.